packages/react-ui/src/widgets/utils/chartUtils.js

```diff
@@ -68,3 +68,25 @@ export function getChartSerie(chart, index) {
 
   return { option, serie };
 }
+
+export function getChartData(chart) {
+  let data = [];
+  if (chart) {
+    const option = chart.getOption();
+    data = option.series[0]?.data;
+  }
+  return data;
+}
+
+export function calculateTextSize(text, fontSize = 8) {
+  let span = document.getElementById('computedTextWidth');
```

Comment:
this search by id doesn't look flexible for a utils function, pass id from outside as param?.

Also, the name of the function doesn't look clear to me. It doesn't calculate text size, it basically deals with styles for an element or creates it. Please try to give this a better name... @eamador